src/bin/lfortran.cpp

```diff
@@ -1805,6 +1808,7 @@ int main(int argc, char *argv[])
         app.add_flag("-g", compiler_options.emit_debug_info, "Compile with debugging information");
         app.add_option("-D", compiler_options.c_preprocessor_defines, "Define <macro>=<value> (or 1 if <value> omitted)")->allow_extra_args(false);
         app.add_flag("--version", arg_version, "Display compiler version information");
+        app.add_option("--rpath", extra_runtime_lib_path, "Library path to be used during linking");
```

Comment:
```suggestion
        app.add_option("-W", extra_rpaths, "Linker flags")->allow_extra_args(false);
```

See https://github.com/lfortran/lfortran/issues/1998#issuecomment-1644953716.